app/streamSelector/browser.js

```diff
@@ -0,0 +1,87 @@
+const { desktopCapturer, ipcRenderer } = require("electron");
+window.addEventListener("DOMContentLoaded", () => {
+  ipcRenderer.once('get-screensizes-response', (event, screens) => {
+    let windowsIndex = 0;
+    const sscontainer = document.getElementById('screen-size');
+    screens.forEach((s, i) => {
+      const opt = document.createElement('option');
+      opt.appendChild(document.createTextNode(s.name));
+      opt.value = i;
+      sscontainer.appendChild(opt);
+    });
+    let defaultSelection = screens.findIndex(s => {
+      return s.default;
+    });
+
+    defaultSelection = defaultSelection > -1 ? defaultSelection : screens.length - 1;
+    sscontainer.selectedIndex = defaultSelection;
+
+    const _tabClick = (e) => {
```

Comment:
probably move this _tabClick and _closePreviews out of this function (out of the window.addEventListener). To make this function smaller and easier to read/maintain/etc